experimental/dds/tree2/src/feature-libraries/modular-schema/modularChangeCodecs.ts

```diff
@@ -146,19 +154,43 @@ function makeV0Codec(
 		return decodedChange;
 	}
 
+	function encodeBuilds(builds: ModularChangeset["builds"]): EncodedBuilds | undefined {
+		if (builds === undefined) {
+			return undefined;
+		}
+		const encoded: EncodedBuilds = nestedMapToFlatList(builds).map(([r, i, t]) =>
+			r !== undefined ? [r, i, t] : [i, t],
```

Comment:
This feels like a potentially unnecessary optimization. Maybe at least add a quick comment either here or in the format file about this optimization.